core/runtime/jsrealm.rs

```diff
@@ -181,8 +187,6 @@ impl JsRealmInner {
     state.exception_state.prepare_to_destroy();
     std::mem::take(&mut *state.js_event_loop_tick_cb.borrow_mut());
     std::mem::take(&mut *state.js_wasm_streaming_cb.borrow_mut());
-    // The OpCtx slice may contain a circular reference
-    std::mem::take(&mut *state.op_ctxs.borrow_mut());
```

Comment:
This needs to be verified with care - I believe this is fine, but not 100% sure. Probably the best to run this against `deno` repo and see if it causes any failures.

Reply:
Actually I think this is fine

Reply:
My only worry here is that you might end up failing to drop the OpDriver in time, which may leave some pending ops that tend to cause trouble.

We can probably add a shutdown method to the driver for this case.